src/components/card/card.jsx

```diff
@@ -12,11 +12,22 @@ import * as styles from "./card.module.scss";
 const Card = ({ data }) => {
   const categoryBlog = useCategory(data.frontmatter.category);
 
+  const urlImages = () => {
+    if (typeof window !== 'undefined' && window.location.href === 'http://localhost:8000/') {
+      return data.frontmatter.thumbnail;
+    } else {
+      if (typeof window !== 'undefined') {
+        const imageUrl = window?.location?.href.slice(0, -6);
+        return `${imageUrl}${data.frontmatter.thumbnail}`
+      }
+    }
+  }
```

Comment:
what will be the url if all conditions are false?

Reply:
Done